README.md

```diff
@@ -183,7 +171,37 @@ to interact with Coq.
 Towards this goal, `coq-lsp` extends and will eventually replace `coq-serapi`,
 which has been used by many to that purpose.
 
-### Planned features
+## FAQ
+
+See our [list of frequently-asked questions](./etc/FAQ.md).
+
+## Troubleshooting and Known Problems
+
+- Some problems can be resolved by restarting `coq-lsp`, in Visual Studio Code,
+  `Ctrl+Shift+P` will give you access to the `coq-lsp.restart` command.
+- In VSCode, the "Output" window will have a "Coq LSP Server Events" channel
+  which should contain some important information; the content of this channel
+  is controlled by the `Coq LSP > Trace: Server` option.
+- If you install `coq-lsp` simultaneously with VSCoq, VSCode gets confused and
+  neither of them may work. `coq-lsp` will warn about that. If you know how to
+  improve this, don't hesitate to get in touch with us.
+- VS Code may send request completions with a stale document, this will be fixed
+  in a new upstream release, c.f. https://github.com/ejgallego/coq-lsp/issues/273
+
+### Working With Multiple Files
+
+`coq-lsp` can't work with more than one file at the same time, due to problems
+with parsing state management upstream. This was fixed in Coq `master` branch
+(to become **Coq 8.18**).
+
+As this is very inconvenient, we do provide a fixed Coq branch that you can
+install using `opam pin`:
+
+- For Coq 8.17:
+  ```opam pin add coq-lsp https://github.com/ejgallego/coq.git#v8.17+lsp`
+- For Coq 8.16: `opam pin add coq-lsp https://github.com/ejgallego/coq.git#v8.16+lsp`
```

Comment:
Seems this should be with 3 backticks?

Reply:
Huh my local changes didn't get pushed, let me fix that.